searx/engines/www1x.py

```diff
@@ -19,8 +19,8 @@
 categories = ['images']
 paging = False
 
-# search-url
-base_url = 'http://1x.com'
+# search-url, no HTTPS
```

Comment:
I think the comment is wrong, because this engine would use HTTPS after merging that pull request.